src/questions/introduction.yaml

```diff
@@ -0,0 +1,170 @@
+- uuid: 16c069e0-b573-4642-9549-925d8bd88df7
+  type: SingleChoice
+  content: What is the difference between traditional Proof-of-Stake (P systems and liquid staking?
```

Comment:
Liquid Staking is a form of Proof-of-Stake. The question should be "what is the difference between Self Staking  and Liquid Staking"